content/numpy/concepts/built-in-functions/terms/trace/trace.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: '.trace()'
+Description: 'Calculates the sum of the elements along the main diagonal of an array.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'Matrices'
+  - 'NumPy'
```

Comment:
```suggestion
Tags:
  - 'Arrays'
  - 'Functions'
  - 'Matrices'
  - 'NumPy'
```
Include only those tags that are present in the [tags.md](documentation/tags.md) list and add one if it is not available in the list, Here `Functions` is not present in the tags.md list you can add it first in the tags.md file.

Reply:
Hi @SaviDahegaonkar I believe the Functions tag is in the `tags.md`